src/OrbitGl/include/OrbitGl/GraphTrack.h

```diff
@@ -51,15 +49,15 @@ class GraphTrack : public Track {
   [[nodiscard]] bool IsCollapsible() const override { return true; }
   [[nodiscard]] bool IsEmpty() const override { return series_.IsEmpty(); }
 
-  virtual void AddValues(uint64_t timestamp_ns, const std::array<double, Dimension>& values) {
+  virtual void AddValues(uint64_t timestamp_ns, absl::Span<const double> values) {
     series_.AddValues(timestamp_ns, values);
   }
 
   [[nodiscard]] uint64_t GetMinTime() const override;
   [[nodiscard]] uint64_t GetMaxTime() const override;
 
-  void SetSeriesColors(const std::array<Color, Dimension>& series_colors) {
-    series_colors_ = series_colors;
+  void SetSeriesColors(absl::Span<const Color> series_colors) {
+    series_colors_ = std::vector(series_colors.begin(), series_colors.end());
```

Comment:
In a setter (or any sink function for that matter) using the vector directly is still the best option, because if the caller already has a vector they can choose to move it instead of making an avoidable copy.

```suggestion
  void SetSeriesColors(std::vector<Color> series_colors) {
    series_colors_ = std::move(series_colors);
```

Reply:
Done! Thanks!